s3router.js

```diff
@@ -25,9 +25,9 @@ function S3Router(options) {
 
     /**
      * Redirects image requests with a temporary signed URL, giving access
-     * to GET an image.
+     * to GET an upload.
      */
-    router.get(/\/img\/(.*)/, function(req, res) {
+    router.get(/\/upload\/(.*)/, function(req, res) {
```

Comment:
Could do a regex, or just have a separate `router.get` handler with a method that they both call.

The regex would be `/\/(?:img|uploads)\/(.*)/` though.  The `?:` means "don't consider this a capturing group", so that `req.params[0]` still references the image path.  Without the `?:`, we'd need `req.params[1]` on line 33.